packages/cozy-client/src/StackLink.js

```diff
@@ -89,19 +89,19 @@ export default class StackLink extends CozyLink {
     this.stackClient = null
   }
 
-  async request(operation, result, forward) {
-    if (this.isOnline && !(await this.isOnline())) {
-      return forward(operation)
+  async request(operation, options, result, forward) {
+    if (this.isOnline && !(await this.isOnline()) && !options?.forceStack) {
```

Comment:
It might be worth it to reverse the conditions here, by evaluation `forceStack` first:  if the `options?.forceStack` is true, there is no need to call the `this.isOnline()`, and thus we save a network call